src/BlockchainTestsFiller/bcUncleTest/uncleHeaderAtBlock2MetropolisFiller.json

```diff
@@ -28,17 +28,6 @@
 				"nonce" : "1"
 			    }
 		    }
-	    },
-	    {
-		    "network" : ["Constantinople"],
-		    "result" : {
-			    "095e7baea6a6c7c4c2dfeb977efac326af552d87" : {
```

Comment:
I don't think so.  It stopped working because I changed the block reward and the difficulty calculation.